### PR TITLE
generate_from_arb has been moved to intl_translantion library

### PR DESCRIPTION
**Pull request by dbacinski**
Scripts used for generating translation files  has moved from `intl` to `intl_translantion` library.

Updated read me file.

### CODE REVIEW

**Comment by dbacinski** [comment]
I signed it!

**Comment by Hixie** [comment]
Does that work? `intl_translantion` looks like a typo...

**Comment by dbacinski** [comment]
Fixed typo and tested.

I have successfully regenerated language files and run Stocks app.

File: examples/stocks/pubspec.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ dependencies:
   flutter:
     sdk: flutter
   intl: '>=0.14.0 <0.15.0'
+  intl_translation:
```
Comment (abarth) [inline examples/stocks/pubspec.yaml]: Don't we need a version constraint here?

**Comment by dbacinski** [comment]
Added version.

Not sure how to force Travis to download Flutter/Dart dependencies and make build to succeed .

**Comment by abarth** [comment]
> Not sure how to force Travis to download Flutter/Dart dependencies and make build to succeed .

We have several Dart packages, and we have an extra check on Travis to make sure they all use a consistent version of all their transitive dependencies.  What Travis is saying is that we're now pulling in version 0.2.3 of the `isolate` package in stocks but other packages in the repo are pulling in version 1.0.0.

My guess is the easiest thing will be to add a version constraint in the stocks pubspec.yaml to require version 1.0.0 of `isolate` package.  Usually pub gives you the most recent allowed version, so you might find a version conflict with another package when you do that.

**Comment by dbacinski** [comment]
Success. Thank you!

**Comment by abarth** [comment]
Right on.  Thanks!

**Comment by abarth** [comment]
LGTM

**Pull request merged**